aten/src/ATen/Declarations.cwrap

```diff
@@ -131,7 +131,7 @@
   return: THTensor*
   arguments:
     - THTensor* self
-    - arg: IntListSize size
```

Comment:
Is it intentional that you also changed `IntListSize` to `IntListRefSize`?

Reply:
It doesn't really matter, this is just some internal thing to distinguish between sizes and strides for scalar checks.  I'd say `IntListRefSize` is slightly better.